.github/workflows/rust.yml

```diff
@@ -63,10 +63,10 @@ jobs:
         run: cargo build --release
 
       - name: Prepare for publish
-        run: cp -r src rust_bindings/cpp
+        run: ln -s src rust_bindings/cpp
 
       - name: Publish to crates.io (dry run)
-        run: cargo publish --dry-run -p chiavdf
+        run: cargo publish --dry-run -p chiavdf --allow-dirty
```

Comment:
I think a comment explaining `--allow-dirty` would be a good idea

Reply:
Added the comments